build/commands/lib/config.js

```diff
@@ -392,8 +392,14 @@ Config.prototype.buildArgs = function () {
     args.cc_wrapper = path.join(this.nativeRedirectCCDir, 'redirect_cc')
   }
 
-  if (this.targetArch === 'x86' && process.platform === 'linux') {
-    // Minimal symbols for target Linux x86, because ELF32 cannot be > 4GiB
+  if ((this.targetArch === 'x86' && process.platform === 'linux') ||
+      this.getTargetOS() === 'win') {
```

Comment:
this should be for x86 only.

Reply:
Are you sure? It happened for ARM64 and is now happening for x86. Seems like it is only a matter of time until it also happens for x64.

Reply:
I don't know how bad `symbol_level=1` on windows would be. If we can keep higher value, it should be easier to debug crashes. So I prefer to keep it if possible for x64.